pandas/core/indexes/multi.py

```diff
@@ -2328,23 +2337,41 @@ def convert_indexer(start, stop, step, indexer=indexer, labels=labels):
             j = labels.searchsorted(loc, side='right')
             return slice(i, j)
 
-    def get_locs(self, tup):
+    def get_locs(self, seq):
         """
-        Given a tuple of slices/lists/labels/boolean indexer to a level-wise
-        spec produce an indexer to extract those locations
+        Get location for a given label/slice/list/mask or a sequence of such as
+        an array of integers.
 
         Parameters
         ----------
-        key : tuple of (slices/list/labels)
+        seq : label/slice/list/mask or a sequence of such (one for each used
+              level. If a level should not be used, set it to
+              ``slice(None)``).
```

Comment:
Multi-line type specifications don't really work (does not render correctly in sphinx, unless you use `\` to indicate line continuation). But you can also spit this up:

```
seq : label/slice/list/mask or a sequence of such (one for each level)
    More explanation ...
```

Reply:
changed